transport/src/test/java/me/uport/sdk/transport/ResponseParserTest.kt

```diff
@@ -38,19 +42,206 @@ class ResponseParserTest {
     )
 
     @Test
-    fun `parse jwt from URL`() {
-        valid.forEach { redirect ->
+    fun `parse jwt from URL with access_token key`() {
+        validAccessTokenTypeURLs.forEach { redirect ->
             val token = ResponseParser.extractTokenFromRedirectUri(redirect)
             assert(token).isNotNull()
         }
     }
 
-    @get:Rule
-    val expectedExceptionRule: ExpectedException = ExpectedException.none()
+    @Test
```

Comment:
I was a bit conflicted on how to approach testing here.
I could easily have just added new URLs to the previous tests but I feel this approach will make it easier when we need to debug or resolve any breaking changes

Reply:
Good call to split them up by use case